src/styles/Colors.scss

```diff
@@ -1,6 +1,7 @@
 $error: #ff4d4dd5;
 $lightBlue: #c4cfdb;
 $linkedinBlue: #3f51b5;
+$linkedinLightBlue: #3f51b512;
```

Comment:
I think it should be #3f51b5 (from my research #3f51b512 doesnt exist)

Reply:
The last 2 digits are transparent.